language/id.yml

```diff
@@ -1,63 +1,311 @@
+fed-check-identity: Tolong tanyakan kepada pemilik federasi untuk konfirmasi tindakan ini dengan mengklik tombol dibawah ini.
+fed-log-set-group: "Grup ini diatur sebagai **{name}** catatan"
+fed-log-set-chnl: "Saluran ini telah ditetapkan sebagai catatan federasi untuk **{}**\nall tindakan terkait federasi akan dicatat di sini."
+fed-invalid-identity: Pengguna yang mengklik tombol konfirmasi bukanlah pemilik federasi.\nHanya pemilik federasi yang dapat menetapkan catatan federasi.
+fed-log-unset: Catatan federasi dari **{}** telah dibatalkan.
+fed-autoban: "Pengguna {} dilarang dalam federasi ini (**{}**), dan sudah dikeluarkan.\n**Reason:** {}\n**Banned on:** {}"
+#endregion
+
 #region language
 language-button: Bahasa
 language-help: |
   Bot ini hadir dengan dukungan multi bahasa.
-  Anda dapat memilih preferensi bahasa Anda!\n
+  Anda dapat memilih preferensi bahasa anda!\n
   ** Bahasa yang tersedia: **
   - 🇺🇸 Bahasa Inggris.
-  - 🇮🇩 Indonesia.\n
+  - 🇮🇩 Bahasa Indonesia.\n
   Perintah:
-  × /setlang | /lang | /language : setel bahasa yang anda inginkan.\n
+  × /setlang | /lang | /language : Mengganti bahasa yang saya gunakan.\n
   **Catatan**
-  Anda dapat menggunakan perintah ini pada grup untuk mengubah pengaturan bahasa pada grup itu.\n
+  Anda dapat menggunakan perintah ini pada grup anda untuk mengganti bahasa yang digunakan.\n
   Bantu kami mencakup lebih banyak bahasa dengan berkontribusi di [Crowdin!] (https://crowdin.com/project/anjani-bot)
-current-language: "Bahasa **{}** saat ini adalah **{}**\nPilih bahasa di bawah ini, untuk mengganti"
-language-code-error: Telah terjadi kesalahan (tidak ada string bahasa yang tersedia)... tolong hubungi grup dukungan
-language-set-succes: Bahasa berhasil disetel ke {}
-language-invalid: Kode bahasa tidak cocok.\nKode bahasa yang tersedia {}
+current-language: "Bahasa **{}** saat ini adalah **{}**\nKlik bahasa yang ingin kamu gunakan"
+language-code-error: Telah terjadi kesalahan (tidak ada string bahasa yang tersedia) Silahkan hubungi grup support saya.
+language-set-succes: Bahasa yang digunakan sudah berhasil dirubah ke {}
+language-invalid: Mohon maaf kode bahasa tidak cocok.\nKode bahasa yang tersedia {}
 #endregion
+
+
+
 #region misc
 miscs-button: Lainnya
+miscs-help: |
+ Berisi perintah kecil dan sederhana yang mungkin tidak cocok di mana pun.\n
+  **Perintah:**
+  ×  /id : Menampilkan ID pengguna, ID pesan, ID grup dan banyak lagi.
+  ×  /info: Menampilkan informasi tentang seorang pengguna.
+  ×  /ping : Menampilkan latensi bot.
+  ×  /paste <flags> : Menyalin dan menempel file teks ke layanan pastebin.
+      flags:
+          [-d, -dogbin] = menggunakan dogbin (default)
+          [-n, -neko, -nekobin] = menggunakan nekobin
+          [-h, -haste, -hastebin] = menggunakan hastebin\n
+  **Perintah Spesial:**
+  ×  /covid <country> : Menampilkan informasi Covid-19 di suatu negara.
 covid-text: |
   **Statistik Virus Corona di {country}**
   **pada {date}**\n
```

Comment:
Can you delete this Covid related string since we dropped it after rebasing the core.

Reply:
Ok sure.